From 9ea6b88ba35d2b41c3a291f74862524e9d4e437e Mon Sep 17 00:00:00 2001
From: Marek Vasut <marex@denx.de>
Date: Mon, 31 Oct 2016 20:09:41 +0100
Subject: [PATCH 09/15] ARM: at91: ma5d4: Enable random ethaddr

Use random ethaddr by default in case no ethaddr is set.

Signed-off-by: Marek Vasut <marex@denx.de>
---
 configs/ma5d4evk_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/ma5d4evk_defconfig b/configs/ma5d4evk_defconfig
index e2a68fb..286c713 100644
--- a/configs/ma5d4evk_defconfig
+++ b/configs/ma5d4evk_defconfig
@@ -14,6 +14,7 @@ CONFIG_VERSION_VARIABLE=y
 # CONFIG_DISPLAY_BOARDINFO is not set
 CONFIG_SPL=y
 CONFIG_HUSH_PARSER=y
+CONFIG_NET_RANDOM_ETHADDR=y
 CONFIG_CMD_BOOTZ=y
 # CONFIG_CMD_IMI is not set
 # CONFIG_CMD_IMLS is not set
-- 
2.10.2

